opentech/apply/funds/views.py

```diff
@@ -373,7 +373,7 @@ def form_valid(self, form):
             project=project,
         )
 
-        return redirect('apply:submissions:detail', submission.id)
+        return redirect('apply:projects:detail', project.id)
```

Comment:
Could update the `get_success_url` method, but not too fussed. Depends on the outcome of any changes to #1353

Reply:
Fixed in 6c7dd74b, which adds `Project.get_absolute_url()` so that now redirects to the Project detail page.